src/identity/SolidOidcIdentityProvider.ts

```diff
@@ -0,0 +1,87 @@
+import { Provider } from 'oidc-provider';
+
+/**
+ * An Extension of node-oidc-provider with configurations specifically for
+ * Solid OIDC
+ */
+export default class SolidOidcIdentityProvider extends Provider {
+  public constructor() {
+    // TODO (@jackson) [>1.0.0]: finish this.
+    // Also decide which of these should be configurable
```

Comment:
All; let's pass everything through dependency injection. There is no need to have any declarative configuration in code files.

Reply:
Sounds good